README.rst

```diff
@@ -58,38 +58,94 @@ All you need to do is be able to generate a JSON structure that includes your AP
 Install
 -------
 
-
 First you need to install autoapi:
 
 .. code:: bash
-	
-	pip install sphinx-autoapi
+
+    pip install sphinx-autoapi
 
 Then add it to your Sphinx project's ``conf.py``:
 
 .. code:: python
 
-	extensions = ['autoapi.extension']
+    extensions = ['autoapi.extension']
 
-        # Document Python Code
-	autoapi_type = 'python'
-	autoapi_dir = 'path/to/python/files'
+    # Document Python Code
+    autoapi_type = 'python'
+    autoapi_dirs = ['path/to/python/files', 'path/to/more/python/files']
 
-        # Or, Document Go Code
-	autoapi_type = 'go'
-	autoapi_dir = 'path/to/go/files'
+    # Or, Document Go Code
+    autoapi_type = 'go'
+    autoapi_dirs = 'path/to/go/files'
 
-AutoAPI will automatically add itself to the last TOCTree in your top-level ``index.rst``.
+AutoAPI will automatically add itself to the last TOCTree in your top-level
+``index.rst``.
 
-This is needed because we will be outputting rst files into the ``autoapi`` directory.
-This adds it into the global TOCTree for your project,
-so that it appears in the menus.
+This is needed because we will be outputting rst files into the ``autoapi``
+directory.  This adds it into the global TOCTree for your project, so that it
+appears in the menus.
 
-We hope to be able to dynamically add items into the TOCTree, and remove this step.
-However, it is currently required.
+We hope to be able to dynamically add items into the TOCTree, and remove this
+step.  However, it is currently required.
 
 See all available configuration options in :doc:`config`.
 
+Setup
+-----
+
+.NET
+~~~~
+
+The .NET mapping utilizes the tool `docfx`_. To install ``docfx``, first
+you'll need to `install a .NET runtime on your system <ASP.NET Installation>`_.
+
+The docfx tool can be installed with::
+
+    dnu commands install docfx
+
+By default, ``docfx`` will output metadata files into the ``_api`` path. You
+can configure which path to output files into using a ``docfx.json`` file in
+your project's repository. Here's an example from the ASP.NET docs:
+
+.. code:: json
+
+    {
+      "metadata": [{
+        "src": [{
+          "files": [
+            "aspnet/identity/src/\*\*/project.json"
+          ],
+          "exclude": [
+            "aspnet/testing/src/Microsoft.AspNet.Testing/\*"
+          ]
+        }],
+        "dest": "docs/_api",
+        "force": "true",
+        "raw": "true"
+      }]
+    }
```

Comment:
I'm a little weary about putting instructions like this in our own docs, as I imagine they will go out of date. Perhaps we should just point at ASP.Net's as an example? Or link to the docfx docs around this?